includes/class-amp-story-post-type.php

```diff
@@ -158,6 +161,11 @@ public static function admin_enqueue_scripts() {
 		if ( self::POST_TYPE_SLUG !== $screen->post_type ) {
 			return;
 		}
+		
+		// Only output scripts and styles where in edit screens.
+		if ( ! in_array( $hook, [ 'post.php', 'post-new.php' ] ) ) {
```

Comment:
What if a Story is not new anymore? E.g. `/wp-admin/post.php?post=123&action=edit`?

Reply:
So this logic means that scripts are displayed on post.php or post-new.php

Reply:
Ah, completely ignored the `post.php`. 🤦‍♀